include/NovelRT.h

```diff
@@ -38,6 +38,10 @@
 #include <Windows.h>
 #endif
 
+#if defined(__APPLE__)
+#include <mach-o/dyld.h>
+#endif
```

Comment:
This is so that we can use Apple's native function to find the executable's file path - this _could_ be moved possibly into `NovelRT::Utilities::Misc`

Reply:
This looks like a system include, so right under the windows.h linkage is fine.